test/lit/passes/inlining-trivial-calls.wast

```diff
@@ -0,0 +1,164 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; Check that "trivial calls" are always inlined, even when optimizing for
+;; size.
+;;
+;; A trivial call is a function that calls another, using its locals in
+;; the order, without skipping any locals.
+;;
+;; These functions can always be inlined because they can't cause binary size
+;; increase at the call sites.
+
+;; RUN: foreach %s %t wasm-opt -all --inlining --shrink-level=0 -S -o - | filecheck %s
+;; RUN: foreach %s %t wasm-opt -all --inlining --shrink-level=1 -S -o - | filecheck %s
+;; RUN: foreach %s %t wasm-opt -all --inlining --shrink-level=2 -S -o - | filecheck %s
+;; RUN: foreach %s %t wasm-opt -all --inlining --shrink-level=3 -S -o - | filecheck %s
```

Comment:
I'm not sure these `RUN` lines do what I want: I want to run the same test with multiple `--shrink-level` parameters and I want to see the same output.

Is this how I use the `RUN` lines for this?

Reply:
Yes, I think this is correct. All these RUNs will use the same CHECKs.